makefile

```diff
@@ -1,23 +1,30 @@
 .PHONY: install uninstall reinstall test clean bsp2svg bsp2wad image2spr pak qmount spr2image unpak unwad wad
 
+prepare:
+	pip3 install -r requirements-dev.txt
+	pip3 install -r requirements.txt
+
+format:
+	python3 -m black ./
```

Comment:
Python 2.7 was deprecated 3 years ago. Do we still need pip3/python3 on Linux machines? On windows both aliases point to the same thing.

Reply:
Yeah, I'm not happy about that change, to be honest. It's like the `python` command is still reserved for version 2.7, at least on Linux.

![image](https://github.com/joshuaskelly/quake-cli-tools/assets/14064112/05776e15-c42f-4e1f-9eb9-4ca1730a0cf8)

Reply:
Okay. It is what is it. 🤷